packages/website/ts/pages/documentation/developers_page.tsx

```diff
@@ -24,11 +44,71 @@ export interface DevelopersPageProps {
 }
 
 export interface DevelopersPageState {
-    isHoveringSidebar: boolean;
-    isHoveringMainContent: boolean;
     isSidebarScrolling: boolean;
 }
 
+const isUserOnMobile = sharedUtils.isUserOnMobile();
+
+const scrollableContainerStyles = `
+    position: absolute;
+    top: ${TOP_BAR_HEIGHT}px;
+    left: 0px;
+    bottom: 0px;
+    right: 0px;
+    overflow-x: hidden;
+    overflow-y: scroll;
+    -webkit-overflow-scrolling: touch;
+    /* Required for hide/show onHover of scrollbar on Microsoft Edge */
+    -ms-overflow-style: -ms-autohiding-scrollbar;
+`;
+
+interface SidebarContainerProps {
+    className?: string;
+}
+
+const SidebarContainer =
+    styled.div <
+    SidebarContainerProps >
+    `
+    ${scrollableContainerStyles}
+    padding-top: 27px;
+    padding-bottom: 100px;
+    padding-left: ${SIDEBAR_PADDING}px;
+    padding-right: ${SIDEBAR_PADDING}px;
+    overflow: hidden;
+    &:hover {
+        overflow: auto;
+        padding-right: ${SIDEBAR_PADDING - SCROLLBAR_WIDTH}px;
+    }
+`;
+
+interface MainContentContainerProps {
+    className?: string;
+}
+
+const MainContentContainer =
+    styled.div <
+    MainContentContainerProps >
+    `
+    ${scrollableContainerStyles}
+    padding-top: 0px;
+    padding-left: 50px;
+    padding-right: 50px;
+    overflow: ${isUserOnMobile ? 'auto' : 'hidden'};
+    &:hover {
+        padding-right: ${50 - SCROLLBAR_WIDTH}px;
+        overflow: auto;
+    }
+    @media (max-width: 40em) {
```

Comment:
this makes me happy. Just wanted to point something out to you: https://github.com/0xProject/0x-monorepo/blob/development/packages/website/ts/style/media.ts

If you're doing media queries, you probably want to use this utility (used as seen here https://github.com/styled-components/styled-components/blob/master/docs/tips-and-tricks.md#media-templates) as it will guarantee we are using the same breakpoints that we are in js (using the ScreenWidths enum).